### PR TITLE
Add samples packaging info

### PR DESCRIPTION
**Pull request by tiembo**
`icon.png` is copied from `shared/src/main/res/mipmap-xxxhdpi/ic_launcher.png`.

### CODE REVIEW

File: .google/packaging.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# GOOGLE SAMPLE PACKAGING DATA
+#
+# This file is used by Google as part of our samples packaging process.
+# End users may safely ignore this file. It has no relevance to other systems.
+---
+status:       PUBLISHED
+technologies: [Android]
+categories:   [Getting Started, Jetpack]
+languages:    [Kotlin]
+solutions:    [Mobile]
```
Comment (benbaxter) [inline .google/packaging.yaml]: And TV!

File: .google/packaging.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# limitations under the License.
+
+# GOOGLE SAMPLE PACKAGING DATA
+#
+# This file is used by Google as part of our samples packaging process.
+# End users may safely ignore this file. It has no relevance to other systems.
+---
+status:       PUBLISHED
+technologies: [Android]
+categories:   [Getting Started, Jetpack]
```
Comment (JoseAlcerreca) [inline .google/packaging.yaml]: it's pretty advanced for a "getting started"

File: .google/packaging.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+languages:    [Kotlin]
+solutions:    [Mobile]
+
+github:       google/iosched
+
+level:        INTERMEDIATE
```
Comment (JoseAlcerreca) [inline .google/packaging.yaml]: I'd say ADVANCED

File: .google/packaging.yaml (hunk 2 above)
Comment (tiembo) [reply .google/packaging.yaml]: Removed.

File: .google/packaging.yaml (hunk 3 above)
Comment (tiembo) [reply .google/packaging.yaml]: Changed

File: .google/packaging.yaml (hunk 1 above)
Comment (tiembo) [reply .google/packaging.yaml]: Added!

**Pull request closed without merging**